packages/debugger/style/breakpoints.css

```diff
@@ -55,25 +55,34 @@
   margin-left: auto;
 }
 
-[data-jp-debugger='true'] .CodeMirror-gutter-wrapper::after {
+[data-jp-debugger='true'] .cm-breakpoint-gutter .cm-gutterElement:empty::after {
   content: '●';
   color: var(--jp-error-color1);
   opacity: 0;
-  position: absolute;
-  left: 3px;
-  top: -1px;
 }
 
-[data-jp-debugger='true'].jp-Editor .CodeMirror-gutter-wrapper:hover::after,
+.cm-gutter {
+  cursor: default;
+}
+
+.cm-breakpoint-gutter .cm-gutterElement {
+  color: var(--jp-error-color1);
+  padding-left: 5px;
+  font-size: 20px;
```

Comment:
Should it be related to code font size?

Reply:
This was an attempt to get the breakpoint aligned. Otherwise, the breakpoint is in the upper left corner.